src/templates/app/user/warning_scam_message.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+const WarningScamMessage = () => (
+    <React.Fragment>
+        <div id='warning_scam_message' className='lightbox invisible'>
+            <div id='warning_scam_message_content'>
+                <h1 id='warning_scam_message_title'>{it.L('Beware of the fake links.')}</h1>
+                <p id='warning_scam_message_subtitle'>{it.L('A link can contain the word "Binary" and still be fake.')}</p>
+                <h2 id='warning_scam_message_title_secondary'>{it.L('Do not get lured to fake "Binary" pages!')}</h2>
+                <p id='warning_scam_message_text_secondary'>{it.L('You may see links to websites with a fake Binary login page where you’ll get scammed for your money. ')}</p>
+                <div className='warning-scam-message-info-message'>
+                    <img className='icon-sm margin-right-0' src={it.url_for('images/pages/warning_scam_message/ic-account-cross.svg')} />
+                    <h3 id='warning_scam_message_scam_title_text'>{it.L('Do not trust and give away your credentials on fake websites, ads or emails.')}</h3>
```

Comment:
```suggestion
        <div id='warning_scam_message' className='lightbox invisible'>
            <div id='warning_scam_message_content'>
                <h1 id='warning_scam_message_title'>{it.L('Beware of the fake links.')}</h1>
                <p id='warning_scam_message_subtitle'>{it.L('A link can contain the word "Binary" and still be fake.')}</p>
                <h2 id='warning_scam_message_title_secondary'>{it.L('Do not get lured to fake "Binary" pages!')}</h2>
                <p id='warning_scam_message_text_secondary'>{it.L('You may see links to websites with a fake Binary login page where you’ll get scammed for your money. ')}</p>
                <div className='warning-scam-message-info-message'>
                    <img className='icon-sm margin-right-0' src={it.url_for('images/pages/warning_scam_message/ic-account-cross.svg')} />
                    <h3 id='warning_scam_message_scam_title_text'>{it.L('Do not trust and give away your credentials on fake websites, ads or emails.')}</h3>
```
Best not to use `ids` if there's no intention of changing the content dynamically from JavaScript